File: 2.server/src/main/java/ru/vitaliy/petrov/server/forms/requests/UserProfileCreationRequest.java
package ru.vitaliy.petrov.server.forms.requests;

import lombok.Data;

@Data
public class UserProfileCreationRequest {
    private final String fullName;
    private final String dateOfBirth;
    private final String residentialAddress;
    private final String placeOfWork;
    private final String positionAtWork;
    private final String workPhoneNumber;
    private final String driverLicenseNumber;
}

File: client/app/src/main/java/com/petrov/vitaliy/caraccidentapp/domain/models/requests/user/UserProfileCreationRequest.kt
package com.petrov.vitaliy.caraccidentapp.domain.models.requests.user

import java.sql.Date

data class UserProfileCreationRequest(
    val fullName: String,
    val dateOfBirth: Date,
    val residentialAddress: String,
    val placeOfWork: String,
    val positionAtWork: String,
    val workPhoneNumber: String,
    val driverLicenseNumber: String
)